steps/appcenter-app-release/1.0.1/step.yml

```diff
@@ -0,0 +1,63 @@
+title: AppCenter App Release
+summary: |
+  Release an application on Microsoft AppCenter
+description: |
+  Release an application on Microsoft AppCenter
+website: https://github.com/fileformat/bitrise-step-appcenter-app-release
+source_code_url: https://github.com/fileformat/bitrise-step-appcenter-app-release
+support_url: https://github.com/fileformat/bitrise-step-appcenter-app-release/issues
+published_at: 2019-08-21T08:57:00-0400
```

Comment:
The date seems to be invalid a : is missing
2019-08-21T08:57:00-04:00 would be the correct.
Did you use stepman share step to create this?